doc/tutorial/tutorial.mdk

```diff
@@ -2203,8 +2203,8 @@ it is shadowed by the local `x`. The other cases are straightforward.
 
 > _Note for experts: Because we will only reduce closed expressions,
   where all variables
-  are bound by previous lambdas, we will ever only substitute closed
-  expressions (closed values in fact), and this naive definition of

```

Comment:
I don't think it's CBV, but the fact that the reduction is weak (i.e. not reducing under binders). A closed term would correctly CBN-reduce with this definition. However if we reduce open terms (say, under binders), we might get captures. We are indeed only substituting with values because of CBV, but that's not what allows for the simplification.

So I believe the original text is correct (except for the "ever only" typo).